<ui version="4.0" >
 <class>TruncationOptionWindow</class>
 <widget class="QDialog" name="TruncationOptionWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Truncation Selection Option</string>
  </property>
  <layout class="QVBoxLayout" name="truncationOptionWindowVLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="sizeIncrement" >
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="title" >
        <string>Choose cycle or group:</string>
       </property>
       <layout class="QHBoxLayout" name="groupBoxHLayout" >
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>6</number>
        </property>
        <property name="bottomMargin" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QComboBox" name="switchComboBox" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <item>
             <property name="text" >
              <string>Cycle</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Group</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="comboboxVLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="cycleComboBox" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="groupComboBox" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TruncationOptionWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TruncationOptionWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
